documentation/puppet_server_metrics.markdown

```diff
@@ -82,6 +81,8 @@ Grafana runs as a web dashboard, and the `grafanadash` module configures it to u
 
 This loads a dashboard with nine graphs that display various metrics exported from the Puppet Server to the Graphite server. (For details, see [Using the Grafana dashboard](#using-the-sample-grafana-dashboard).) However, these graphs will remain empty until you enable Puppet Server's Graphite metrics.
 
+> Note: If you want to integrate Puppet Server's Grafana exporting with your own infrastructure, use the `grafanadash` module. If you want permanent visualization of metrics, use the `puppetlabs-puppet_metrics_dashboard` module. See [Puppet Metrics Collection](https://github.com/puppetlabs/best-practices/blob/master/puppet-enterprise-metrics-collection.md) for more information. 
```

Comment:
I wouldn't use the word "permanent" in "If you want permanent visualization of metrics"